ParisBNF/abb/BNFabb30.xml

```diff
@@ -0,0 +1,720 @@
+<?xml version="1.0" encoding="UTF-8"?><?xml-model href="https://raw.githubusercontent.com/SChAth/schema/master/tei-betamesaheft.rng"
+schematypens="http://relaxng.org/ns/structure/1.0"?><?xml-model href="https://raw.githubusercontent.com/SChAth/schema/master/tei-betamesaheft.rng"
+type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?><TEI xmlns="http://www.tei-c.org/ns/1.0" xml:id="BNFabb30" xml:lang="en" type="mss">
+    <teiHeader xml:base="https://betamasaheft.eu/">
+        <fileDesc>
+            <titleStmt>
+                <title>Books of the Old Testament</title>
+                <editor role="generalEditor" key="AB"/>
+                <editor key="DN"/>
+                <funder>Akademie der Wissenschaften in Hamburg</funder>
+            </titleStmt>
+            <editionStmt>
+                <p/>
+            </editionStmt>
+            <publicationStmt>
+                <authority>Hiob-Ludolf-Zentrum für Äthiopistik</authority>
+                <publisher>Die Schriftkultur des christlichen Äthiopiens: Eine multimediale
+                    Forschungsumgebung / Beta maṣāḥǝft</publisher>
+                <pubPlace>Hamburg</pubPlace>
+                <availability>
+                    <licence target="http://creativecommons.org/licenses/by-sa/4.0/"> This file is
+                        licensed under the Creative Commons Attribution-ShareAlike 4.0. </licence>
+                </availability>
+            </publicationStmt>
+            <sourceDesc>
+                <msDesc xml:id="ms">
+                    <msIdentifier>
+                        <repository ref="INS0303BNF"/>
+                        <collection>Manuscrits orientaux</collection>
+                        <collection>Fonds éthiopien</collection>
+                        <collection>d'Abbadie</collection>
+                        <idno>BnF Éthiopien d'Abbadie 30</idno>
+                        <altIdentifier>
+                        <idno facs="https://archivesetmanuscrits.bnf.fr/ark:/12148/cc99487b">BnF Éthiopien d'Abbadie 35</idno>
+                          </altIdentifier>
+                        </msIdentifier>
+
+
+                          <msContents>
+                          <msItem xml:id="ms_i1">
+                             <locus from="3ra" to="47rc"/>
+                             <title type="complete" ref="LIT1340EnochE"/>
+                             <incipit xml:lang="gez"><hi rend="rub">ዘሄኖክ᎓ ነቢይ᎓ ቃለ᎓ በረከት᎓</hi> ዘከመ᎓ ባረከ᎓
+                               ኅሩ<hi rend="rub">ያነ᎓ ወጻድቃነ᎓ እለ᎓ ሀ</hi>ለዉ᎓ ይኩኑ᎓ በዕለተ᎓ ም<hi rend="rub">
+                               ንዳቤ᎓ ለአሰስሎ᎓ ኲሎ᎓</hi> እኩያን᎓ ወረሲዓን፨</incipit>
+                             <explicit xml:lang="gez">ወየሐውሩ᎓ እሙንቱሂ᎓ በኀበ᎓ ተጽሕፈ᎓ ሎሙ᎓ መዋዕል᎓ ወአዝማን፨</explicit>
+                             <note>Book of Henok. Division into parts is indicated by the word <foreign>ክፍ</foreign> or
+                             <foreign xml:lang="gez">ክፍል</foreign>, rubricated, without any number.
```

Comment:
I believe would be better to mark up the geez words in the text everywhere with `<foreign xml:lang="gez">`

Reply:
Ok.